omod/src/main/java/org/openmrs/module/appointments/web/mapper/AppointmentMapper.java

```diff
@@ -99,6 +99,10 @@ public void mapAppointmentRequestToAppointment(AppointmentRequest appointmentReq
         if (StringUtils.isNotBlank(appointmentRequest.getStatus())){
             appointment.setStatus(AppointmentStatus.valueOf(appointmentRequest.getStatus()));
         }
+
+        if (appointmentRequest.getDateAppointmentScheduled() != null) {
+            appointment.setDateAppointmentScheduled(appointmentRequest.getDateAppointmentScheduled());
+        }
```

Comment:
Should this default to "date created"?

Reply:
the property defaults to new Date() on instantiation. Is that limiting?